scripts/run_examples.py

```diff
@@ -54,6 +52,23 @@ def run_examples(keep_artifacts=None):
     int
         Number of failed examples
     """
+    conda_exes = [None]
+    if platform.system() != 'Windows':
+        env_name = "constructor-testing-micromamba"
+        which_cmd = ["conda", "run", "--name", env_name]
+        if platform.system() == "Windows":
+            which_cmd += ["where", "micromamba"]
+        else:
+            which_cmd += ["which", "micromamba"]
```

Comment:
Something isn't right in the conditions here, it checks for `platform.system()` twice here but it can never run the "Windows" branch.

Reply:
It might also be more portable to use [shutil.which](https://docs.python.org/3/library/shutil.html#shutil.which) for finding the command, unless there is an issue with that on Windows?

Reply:
Ah yes, this is a left over bit from my debugging for https://github.com/conda/constructor/pull/503#issuecomment-1067258156. Fixed.

> It might also be more portable to use [shutil.which](https://docs.python.org/3/library/shutil.html#shutil.which) for finding the command

`micromamba` is installed in a different conda environment so isn't on `$PATH`. I can encode the logic for where the environment is but I thought `conda run --name xxx which micromamba` would be more reliable.

Reply:
I tried installing `micromamba` in the same environment but I run into conflicts between `conda-forge` and `defaults`. As I assume you'd rather stick to using `defaults` putting it in another environment seemed easier. Plus this is "only" test code so this seems good enough while ensuring the `conda-standalone` tests aren't behaving differently.

Reply:
Got it, this sounds fine by me.